test/parallel/test-fork-send-returns-boolean.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+const common = require('../common');
+const assert = require('assert');
+const fork = require('child_process').fork;
+
+const n = fork(common.fixturesDir + '/empty.js');
+
+const rv = n.send({ hello: 'world' });
+assert.strictEqual(rv, true);
```

Comment:
I wonder if we have some kind of regression in `-e`:

```
$ out/Release/node -e 'require("child_process").fork("test/fixtures/empty.js").on("exit", console.log)'
# doesn't print, doesn't exit
```

`parallel/test-child-process-fork3` does the same thing from a file and works fine.